ftw/linkchecker/linkchecker.py

```diff
@@ -0,0 +1,100 @@
+from multiprocessing import Pool
+import time
+import requests
+import urls_testing
+import os
+import sys
+
+urls = urls_testing.urls
+number_of_processes = len(urls)
+timeout = 1
+
+
+def millis():
+    return int(round(time.time() * 1000))
+
+
+def get_uri_responses(urls):
+    headers = {
+        'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.102 Safari/537.36'
+    }
+    error = None
+    response = None
+    start_time = millis()
+    try:
+        response = requests.head(urls.get('destination'),
+                                 timeout=timeout,
+                                 headers=headers,
+                                 allow_redirects=False)
+    except requests.exceptions.Timeout:
+        error = 'Timeout'
+    except requests.exceptions.TooManyRedirects:
+        error = 'Too many redirects'
+    except requests.exceptions.ConnectionError:
+        error = 'Connection Error'
+    except requests.exceptions.RequestException as e:
+        error = e
+    time = millis() - start_time
+    result = {
+        'origin': urls.get('origin'),
+        'destination': urls.get('destination'),
+        'data': response,
+        'error': error,
+    }
+    result.update({'time': time})
+
+    return result
+
+
+def limit_cpu():
+    try:
+        sys.getwindowsversion()
+    except AttributeError:
+        isWindows = False
+    else:
+        isWindows = True
+
+    if isWindows:
```

Comment:
Do we need windows support?

Reply:
My idea here was, that if we're create an open source package on pypi, it would be nice to support most systems. Also I didn't know exactly what servers it will run on in future.

Reply:
Well that's honourable. But Plone is hard to run on windows and we cannot (or don't) have the time to put it on windows too. Please postpone Windows support to the next major release.

Reply:
👍 I removed the windows part.